src/app/teams/teams-view.component.ts

```diff
@@ -277,10 +277,20 @@ export class TeamsViewComponent implements OnInit, OnDestroy {
   }
 
   addMembers(selected: any[]) {
-    this.dialogsLoadingService.start();
     const newMembershipDocs = selected.map(
       user => this.teamsService.membershipProps(this.team, { userId: user._id, userPlanetCode: user.planetCode }, 'membership')
     );
+    const checkActiveRequest = this.requests.some((req: any) => {
+      if (req.userId === newMembershipDocs[0].userId) {
```

Comment:
This only affects the first member selected to be added to the team.  All other request docs will remain and only the first member selected will be added.